README.md

```diff
@@ -41,20 +41,21 @@ wget -qO- https://raw.githubusercontent.com/aklivity/zilla-examples/main/startup
 ```
 
 ```text
-Usage: startup.sh [-km][-h KAFKA_HOST -p KAFKA_PORT][-d WORKDIR][-v VERSION][--no-kafka][--auto-teardown][--redpanda] example.name
+Usage: startup.sh [-km][-h KAFKA_HOST -p KAFKA_PORT][-d WORKDIR][-v ZILLA_VERSION][-e EX_VERSION][--no-bootstrap][--redpanda] example.name
 
 Operand:
     example.name          The name of the example to use                                 [default: quickstart][string]
 
 Options:
+    -e | --ex-version     Sets the examples version to download                              [default: latest][string]
     -d | --workdir        Sets the directory used to download and run the example                             [string]
     -h | --kafka-host     Sets the hostname used when connecting to Kafka                                     [string]
     -k | --use-helm       Use the helm install, if available, instead of compose                             [boolean]
     -m | --use-main       Download the head of the main branch                                               [boolean]
     -p | --kafka-port     Sets the port used when connecting to Kafka                                         [string]
-    -v | --version        Sets the version to download                                       [default: latest][string]
+    -v | --zilla-version  Sets the zilla version to use                                      [default: latest][string]
          --auto-teardown  Executes the teardown script immediately after setup                               [boolean]
-         --no-kafka       The script wont try to start a kafka broker                                        [boolean]
+         --no-bootstrap   The script wont try to bootstrap the kafka broker                                  [boolean]
```

Comment:
Suggest `--no-kafka-init` instead of `--no-bootstrap` since we already use the term `bootstrap` to mean something else in `zilla.yaml` and also not `kafka` `bootstrap` server.